### ISSUE TITLE
Exception when saving a Generalisation Set in 1.0.0

### ISSUE THREAD

**Issue by perglr** [OpenPonk/ontouml-profile#3]
![2018-10-28-102512_1884x970_scrot](https://user-images.githubusercontent.com/4477111/47614188-047f3900-da9c-11e8-86d5-26ebb0ea4ab9.png)

[image](https://drive.google.com/a/fit.cvut.cz/file/d/1EpUCuU58MMFgIJIOvCBZSq1giUL-r4eQ/view?usp=drivesdk)

**Comment by JanBliznicenko** [comment]
True. Fortunately, version 1.0.0 is lost to time and it works in the current version
